.renovaterc.json

```diff
@@ -77,6 +77,10 @@
         "managerBranchPrefix": "linting-"
       }
     },
+    {
+      "packagePatterns": ["rrule"],
```

Comment:
I don't actually know if this is valid renovate bot config or how to test it out, but the intention is to have rrule opt out of auto-updates since they could be breaking. I'll leave a comment (pretty sure renovate supports json with comments).